.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,91 @@
+{
+    "configurations": [
+        {
+            "name": "Mac",
+            "includePath": [
+                "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1",
+                "/usr/local/include",
+                "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/9.0.0/include",
```

Comment:
can we avoid hardcoding the clang version number here?

Reply:
This is likely a good default, it will rarely be different - and the cases where it might be relevant (really odd cases), it should be easy to modify for anyone that cares.

Reply:
Is it possible to override these paths in a non-version controlled file? If someone does need to modify these, it would be quite annoying to always have this file show up as modified in the working tree.

Reply:
the `.vscode` directory is by default in `.gitignore`, so any change there will not be version-controlled. I had to `git add -f` to get it in.